.cypress/utils/event_analytics/constants.js

```diff
@@ -54,29 +54,29 @@ export const TEST_QUERIES = [
   },
 ];
 
+export const aggregationValues = [
+  'count',
+  'sum',
+  'avg',
+  'max',
+  'min',
+  'var_samp',
+  'var_pop',
+  'stddev_samp',
+  'stddev_pop',
+];
+
 export const TESTING_PANEL = 'Mock Testing Panels';
 export const SAVE_QUERY1 = 'Mock Flight Events Overview';
 export const SAVE_QUERY2 = 'Mock Flight count by destination';
 export const SAVE_QUERY3 = 'Mock Flight count by destination save to panel';
 export const SAVE_QUERY4 = 'Mock Flight peek';
 
-export const aggregationValues = [
-  'COUNT',
-  'SUM',
-  'AVERAGE',
-  'MAX',
-  'MIN',
-  'VAR_SAMP',
-  'VAR_POP',
-  'STDDEV_SAMP',
-  'STDDEV_POP',
-];
-
 export const querySearch = (query, rangeSelected) => {
```

Comment:
As a code hygiene concern, we try to keep constants and functions (helper functions) in separate files.
This separation helps b/c it's easier to scan the files of "like kind" code.   The two files might change at different paces (yes, this is testing, both are likely slow-moving files)... the churn could be less with the separation.

Even though a lot of the code in `utils/event_analytics/constants.js` was pre-existing, this would be a convenient time to re-organize the file (split).

Reply:
Good catch, separated all helper functions and removed from the constants file.